website/utils/styleConstants.ts

```diff
@@ -0,0 +1,3 @@
+// Layout Constants
+export const largeWidth = 700;
+export const largestWidth = 1138;
```

Comment:
the name of this file and these variables don't indicate where these constants should be used

Reply:
What do you mean by that?

Reply:
when I see these I think to myself "largest width of what?"

Reply:
Would `largestLayoutWidth` make more sense to you?

Reply:
It's referring to the width of the main page container

Reply:
it then raises the question "layout of what?" 🤔

Reply:
The amount of space that component pages are allowed to take up, which changes depending on the size of the viewport

Reply:
They're just pretty random-seeming numbers that were used in a bunch of places so I thought it'd be easier to document here, but very open to other suggestions

Reply:
>It's referring to the width of the main page container

wrote my last reply before I saw this comment. seeing `mainPageContainerWidth.largest` would be a lot more descriptive.

Reply:
its not just the main page, can I do pageContainerWidth.largest

Reply:
two more questions:
- shouldn't this be under the website/styles/ directory
- "large" and "largest" are relative terms. what determines when something is "large"?

Reply:
I can put this under styles. I wasnt sure if it should live there or not because its not using emotion directly

Reply:
The problem is, the large/largest don't map directly to our breakpoints. They're more values relative to one and other than anything else. When do we allow content to take up a large space versus when do we allow content to take up the largest space. There's a part of the CodeDocs page that employs both at the same breakpoint.

Reply:
from what I can tell, it seems that the general rule is that when there are large visual elements that have multiple columns such as the grid or a table that display structured data, then we use "largest", otherwise we use "larger". does that seem right?

Reply:
yes

Reply:
maybe we can call them something like "pageContentWidth.Default" and "pageContentWidth.DataGraphic"